README.md

```diff
@@ -1,105 +1,115 @@
   * [Work flow](#work-flow)
   * [Important code conventions](#important-code-conventions)
   * [Tests](#tests)
-  [Use as JavaScript library](#use-as-javascript-library)
-  [Use as JVM library](#use-as-jvm-library)
+-  [Use as JavaScript library](#use-as-javascript-library)
+-  [Use as JVM library](#use-as-jvm-library)
 
 
 Short intro's
 =============
-For users (bobbin lace makers and designers) with screen-casts:<br>
-https://d-bl.github.io/GroundForge/help/index
+For end users (bobbin lace makers and designers):  
+https://d-bl.github.io/GroundForge/
 
 For developers:
 * `src/scala/main/*` is translated to : `docs/js/GroundForge-opt.js`
 * This is connected client side to HTML with : `docs/js/tiles.js`
-* Translated to JVM for server-side or batch processing, for example something like : `src/test/Demo4Java.java`
+* The source run also in a JVM environment for server-side or batch processing,
+  for example something like : `src/test/Demo4Java.java`
 
 
-DEMO's
+Demo's
 ======
 
 A [dressed up](https://d-bl.github.io/GroundForge/tiles?patchWidth=12&patchHeight=12&a1=ct&b1=ct&c1=ctc&d1=ctc&b2=ctc&d2=ctc&a3=ct&c3=ct&footside=b,-,a,-&tile=831,4-7,-5-&footsideStitch=ctctt&tileStitch=ctc&headsideStitch=ctctt&shiftColsSW=-2&shiftRowsSW=2&shiftColsSE=2&shiftRowsSE=2) version and a dressed down [API](https://d-bl.github.io/GroundForge/API/) version.
-The latter shows some sample code to configure your own web interface or embed just some of the diagrams on your own web page. 
-See also the [API](https://d-bl.github.io/GroundForge/docs/API.md) for other environments than a web-browser.
+The latter shows all possible diagrams with minimal code.
+The first demonstrates interaction with these diagrams
+implemented with limited styling and event handling skills. 
+It evolved into from a proof of concept into the *de facto* user interface.
 
 
-How it's Made / third party data and scripts
-============================================
+Contribute to documentation
+===========================
 
-Proof of concept with D3.js
+GroundForge has various types of documentation.
+
+In this repository
+* Tutorials
+* Pages referred to by info buttons for form fields.
+  Originally intended as tooltips but evolved into too large explanations. 
+  
+Examples of patterns
+* The other repositories listed below
+* Third party blogs and whatever we might not know about
+
+Most common tasks
+-----------------
+* Edit the `.md` files in the `docs` folder.
+* Keep the TOC in the sidebar up to date.
+
+There is a simple [procedure] to propose simple changes to the docs that build the sites.
+Simple changes could be things like typo's, grammar or
+simplified phrasing hoping that automated translators do a better job.
+
+| repository | docs | site | sidebar |
+|------------|------|------|----------|
+| [GroundForge] | [xxx] | [yyy] | [zzz] |
+| [gw-lace-to-gf] | ... | ... | ... |
+| [tesselace-to-gf] | ... | ... | ... |
+| [MAE-gf] | ... | ... | ... |
```

Comment:
@veronika @MAETempels 
Any suggestion how to attack this table?
[preview](https://github.com/d-bl/GroundForge/blob/268cb86b1fde8995f1c22852d65b7d79fbe41ede/README.md)